client/my-sites/themes/sheet.jsx

```diff
@@ -51,6 +53,14 @@ const ThemeSheet = React.createClass( {
 		this.props.dispatch( signup( this.props ) );
 	},
 
+	onDownloadButtonClick() {
+		if ( this.props.download ) {
```

Comment:
This is a tricky question, usually we create we create ZIPs for the themes we convert. In the case of TwentySixteen though, there was no download generated, so I assume when this happens the theme is being directed to be downloaded from WordPress.org.

This can be seen in the Twenty Sixteen showcase page: 

https://theme.wordpress.com/themes/twentysixteen/